workflow_db/database/config/database.yml

```
@@ -4,9 +4,7 @@ login: &login
   password:
   endcoding: utf8
   reconnect: true
-  # socket: /tmp/mysql.sock
   host: 127.0.0.1
-  port: 31313
```

This is intentional because the docker DB is spun up on 31313, and the migrations run against it: https://github.com/LiveRamp/workflow2/blob/master/test_docker/test-db-compose.yml#L9

I kinda suspect this produced an empty schemas jar.  This should probably get cleaned up; I removed the -e in the regenerate script https://github.com/LiveRamp/workflow2/commit/b102af2a5054faf417f34a5f11a7739efb8def2b#diff-ba483b98dca17fb2dba944f6da715f24) because rvm didn't work (https://github.com/rvm/rvm/issues/1198) so it may be silently failing.

---

ok, ty. There's no changes now